src/i18n/locales/en-US.json

```diff
@@ -42,6 +42,10 @@
   "components.firstrun.custompinscreen.title": "Set PIN",
   "components.login.appstartscreen.loginButton": "Login",
   "components.login.custompinlogin.title": "Enter PIN",
+  "components.maintenancemodal.attention": "ATTENTION:",
```

Comment:
Usually better to avoid repeating strings that are already present in the app since it reduces the amount of translation work required